ReactSkia/core_modules/RSkInputEventManager.cpp

```diff
@@ -17,15 +23,30 @@ RSkInputEventManager::RSkInputEventManager(){
                                                               std::placeholders::_2);
   eventId_ = NotificationCenter::defaultCenter().addListener("onHWKeyEvent", handler);
   spatialNavigator_ =  SpatialNavigator::RSkSpatialNavigator::sharedSpatialNavigator();
+  keyRepeat=false;
+  previousKeyType=RNS_KEY_UnKnown;
 }
 
 void RSkInputEventManager::keyHandler(rnsKey eventKeyType, rnsKeyAction eventKeyAction){
   bool stopPropagate = false;
-  if(eventKeyAction != RNS_KEY_Press)
-    return;
+  RNS_LOG_DEBUG("[keyHandler] Key Repeat" << keyRepeat<<"  eventKeyType  " <<eventKeyType << " previousKeyType " <<previousKeyType <<"  eventKeyAction  " << eventKeyAction);
+  if(previousKeyType == eventKeyType  && eventKeyAction == RNS_KEY_Press){
+    keyRepeat = true;
+  }
+  previousKeyType = eventKeyType; 
+  if(eventKeyAction == RNS_KEY_Release ){
+    previousKeyType = RNS_KEY_UnKnown;
+    if(keyRepeat == true){
```

Comment:
is the check required here ?
because under Release action , it need to be by default set to false.

Reply:
In the current implementation, we are not handling the key release. In the case of Press and hold, the key release has to be sent to the textinput.  check is added for that case.